src/lib/components/stepper/components/transact-step.svelte

```diff
@@ -38,23 +42,30 @@
 
   type TransactionWrapperWithGasLimit = TransactionWrapper & { gasLimit: number | undefined };
 
-  interface TransactionTimelineItem {
-    title: string;
-    message: string;
-    txUrl?: string;
-    status:
-      | 'awaitingPrevious'
-      | 'awaitingSignature'
-      | 'submittedToSafe'
-      | 'pending'
-      | 'confirmed'
-      | 'failed'
-      | 'cancelled'
-      | 'rejected'
-      | 'retrying'
-      | 'finalizing'
-      | null;
-  }
+  type TransactionTimelineItem =
+    | {
+        external: false;
```

Comment:
Generic minor suggestion / personal preference for booleans: rename to `isExternal`.